src/Sylius/Behat/Element/Product/ShowPage/VariantsElement.php

```diff
@@ -38,7 +39,7 @@ public function hasProductVariantWithCodePriceAndCurrentStock(
         /** @var NodeElement $variant */
         foreach ($variantRows as $variant) {
             if (
-                $this->hasProductWithGivenNameCodePriceAndCurrentStock($variant, $name, $code, $price, $currentStock)
+            $this->hasProductWithGivenNameCodePriceAndCurrentStock($variant, $name, $code, $price, $channelName, $currentStock)
```

Comment:
```suggestion
                $this->hasProductWithGivenNameCodePriceAndCurrentStock($variant, $name, $code, $price, $channelName, $currentStock)
```
i think that this indentation should stay